releases/tools/release-2.0.0+.md

```diff
@@ -0,0 +1,23 @@
+---
+id: tools-2-0-0
+title: Multiplayer Tools 2.0.0-pre+
+description: Release notes for Multiplayer Tools including new features, updates, bug fixes, known issues, and information to help you upgrade.
+---
+
+## [2.0.0-pre.2] - 2023-05-02
+
+### General
+
+Dropped support for Unity 2020.3; the next supported version is Unity 2021.3
+Fixed `Failed to load type initialization for assembly Unity.Multiplayer.Tools.MetricTypes` runtime exception when building using Managed Stripping level set to high.
+
+### Network Scene Visualization
+
+This release adds the [Network Scene Visualization](../../tools/network-scene-vis.md) to the Multiplayer Tools Package. This tool allows users to visualize networking information (like bandwidth and ownership) on a per-object basis in the scene view using a number of visualizations, including mesh shading and a text overlay.
```

Comment:
Small nit, I think the tool is just called `Network Scene Visualization` rather than `the Network Scene Visualization`

Reply:
At least that's how I've always heard it referred to internally 🙂